docs/_data/components/schemas/array-max-monitored-elements.yaml

```diff
@@ -1,51 +1,73 @@
-# TODO: monitoring-set is no longer referenced
-monitoring-set:
-  description: |
-    Set of transmission facilities noteworthy to reference in queries in TROLIE.  
-    The set of segments included in the monitoring set is implied by their
-    inclusion in the transmission facilities within the monitoring set.  
-  type: object
-  additionalProperties: false
-  properties:
-    id:
-      $ref: ./generic-identifier.yaml
-    description:
-      $ref: '../../openapi-split.yaml#/components/schemas/description'
-    transmission-facilities: &max
-      type: array
-      minItems: 0
-      maxItems: 50000
-      items:
-        $ref: ./generic-identifier.yaml
+forecast-limit-set: &max
+  type: array
+  minItems: 0
+  maxItems: 50000
+  description: Set of forecast limits
+  items:
+    $ref: 'array-max-forecast-periods.yaml#/forecast-limit-item'
 
 forecast-limit-set-detailed:
   <<: *max
   description: Forecast including provenance information.
   items:
     $ref: 'array-max-forecast-periods.yaml#/forecast-limit-item-detailed'
 
-forecast-limit-set:
+named-power-system-objects:
   <<: *max
-  description: Set of forecast limits
+  description: Collection of power system object names
   items:
-    $ref: 'array-max-forecast-periods.yaml#/forecast-limit-item'
+    $ref: ./names.yaml
+
+forecast-rating-proposal:
+  type: object
+  properties:
+    forecast-header:
+      $ref: ./rating-proposal-header.yaml
```

Comment:
If this PR passes muster, I can implement this header concept in the other resources.

Reply:
Agreed.

Reply:
Let's resolve the `object-id` conversation, and I'll make consequential changes from that to just the `forecast-proposal` in this PR. We'll merge that, then I'll open another PR to get the header introduced in the other resources (proposals and snapshots).

Reply:
N.b. in #48 I changed `forecast-header` to `proposal-header` anticipating it can and should be reused across all proposals.